qa-tests-backend/src/main/groovy/services/FeatureFlagService.groovy

```diff
@@ -19,4 +19,12 @@ class FeatureFlagService extends BaseService {
         }
         return flag.enabled
     }
+
+    static boolean isFeatureFlagEnabled(String envVar, boolean defaultVal) {
```

Comment:
How about making it just for scanner? We used to have something similar for PG migration
`isScannerV4()

Reply:
Great suggestion, that is better.  May be removing this all together in the short term as am currently leaning toward disabling Scanner V4 in CI until we figure this out the setup more holistically - should we keep this logic, will make the update.  Thanks @janisz